components/kubernetes/kind-cluster.yaml

```diff
@@ -2,10 +2,15 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
+  extraPortMappings:
+  - containerPort: 30080
+    hostPort: 30080
```

Comment:
Maybe we could benefit from having it configurable. But for now it uses a port that should not be already bound on the host. And I could not see why we'd want to change it